### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by mindys360**
DRAFT
Added CV info. Still needs editing.

### CODE REVIEW

**Comment by mindys360** [comment]
Entered info. Needs editing and formatting.

**Comment by Laura-Novich-OBW** [comment]
Do you want me to merge it as is and you can submit another PR with changes?

File: student-work/2022/Mindy-Steen/ReadMe.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+Run:AI helps organizations streamline and speed up the delivery of AI capabilities through elastic compute infrastructure management and distributed computing.
+
+
+Relocation
+Career Break
+Jan 2019 - Aug 2021
+
+Regional Human Resources Manager
+SEPHORA
+Aug 2015 - Dec 2018 ·
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Mindy-Steen/ReadMe.md:19]:
```suggestion
Aug 2015 - Dec 2018
```

File: student-work/2022/Mindy-Steen/ReadMe.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+United States
+MBA, Business Strategy and Management
+
+Yeshiva University
+BA
+
+Volunteering
+Wise
+Volunteer Judge for research paper competition
+April 2015 - Present
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Mindy-Steen/ReadMe.md:41]:
```suggestion
April 2015 &ndash; Present
```

File: student-work/2022/Mindy-Steen/ReadMe.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+SEPHORA
+Aug 2015 - Dec 2018 ·
+United States
+
+PACSUN
+United States
+National Human Resources Manager (Outlets)
+Jun 2011 - Jun 2012
+Regional Human Resources Manager
+Jun 2009 - Jun 2011
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Mindy-Steen/ReadMe.md:27]:
```suggestion
Jun 2009 &ndash;  Jun 2011
```

File: student-work/2022/Mindy-Steen/ReadMe.md [hunk 4]
```diff
@@ -1 +1,47 @@
...
+
+Regional Human Resources Manager
+SEPHORA
+Aug 2015 - Dec 2018 ·
+United States
+
+PACSUN
+United States
+National Human Resources Manager (Outlets)
+Jun 2011 - Jun 2012
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Mindy-Steen/ReadMe.md:25]:
```suggestion
Jun 2011 &ndash;  Jun 2012
```

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
some small changes, but otherwise a NICE job!

**Comment by mindys360** [comment]
Committed all suggested changes. Thank you.

**Review by Laura-Novich-OBW — APPROVED** [review]
Great job!

**Pull request merged**